Kickstarter-iOS/Views/Controllers/CommentsViewControllerTests.swift

```diff
@@ -41,6 +41,30 @@ internal final class CommentsViewControllerTests: TestCase {
     }
   }
 
+  func testView_WithSuccessFailedRetryingRetrySuccessComments_ShouldDisplayAll() {
```

Comment:
Just double check the screenshot is actually showing the different cell states. Because I had an issue where I was committing empty screenshots. Probably not happening here, but worth mentioning.

Reply:
Yeah, I checked and it was fine.